figures/line.py

```diff
@@ -21,6 +21,10 @@ def _get_line(self, data, key):
             y_axis_label="No. Violations",
         )
         p.line(x, y, legend_label="Progress", line_width=2)
+        try:
+            p.y_range = Range1d(0, (int(max(y)) * 1.1))
+        except ValueError:
+            return
```

Comment:
We can a value error occur? This solution will only cause the ValueError to be masked by a null pointer exception.